lib/forms.ts

```diff
@@ -0,0 +1,92 @@
+'use server'
```

Comment:
Most of this file is about validation and parsing. This is the way NextJS recommends doing this: https://nextjs.org/docs/app/building-your-application/data-fetching/server-actions-and-mutations#server-side-form-validation

Reply:
We used zod here 8a6451e5dcc5d61bb69914f5c4ba1ac457769988, just as in the docs example